sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -839,6 +856,36 @@ export class ServiceBusAtomManagementClient extends ServiceClient {
     if (isUpdate) {
       webResource.headers.set("If-Match", "*");
     }
+
+    const forwardTo = (entityFields as InternalQueueOptions | InternalSubscriptionOptions)
+      .ForwardTo;
+
+    const forwardDeadLetterMessagesTo = (entityFields as
+      | InternalQueueOptions
+      | InternalSubscriptionOptions).ForwardDeadLetteredMessagesTo;
```

Comment:
Looks like there is an extra | here...

Shouldnt `(entityFields as | InternalQueueOptions | InternalSubscriptionOptions)` be `(entityFields as InternalQueueOptions | InternalSubscriptionOptions)`?